pulpito/controllers/job.py

```diff
@@ -1,9 +1,9 @@
 from pecan import expose
 from pecan import conf
 import requests
-import urlparse
-from util import prettify_job
+from urllib.parse import urljoin
```

Comment:
so I assume that we are dropping the py2 support.

Reply:
Yes, we drop it here.